include/openenclave/attestation/attester.h

```diff
@@ -24,6 +24,11 @@
 
 OE_EXTERNC_BEGIN
 
+/**
+ * Bit-wise flags passed to oe_get_evidence() function.
+ */
+#define OE_EVIDENCE_FLAGS_BIT_WRAP_HEADER 0x00000001
```

Comment:
Naming consistency with other flags: e.g. `OE_SGX_FLAGS_DEBUG`, so we should drop the extra `_BIT_`. 

I would also suggest choosing names that describe intent rather than implementation, so `OE_EVIDENCE_FLAGS_EMBED_FORMAT_ID` rather than `WRAP_HEADER`; the same flag may be implemented differently in the future, and we may add more flags to indicate other information we might want to include in the existing header.

Reply:
Agreed!